packages/api/src/memberContext.ts

```diff
@@ -116,23 +116,25 @@ export class MemberContext implements MemberContext {
     userSubscription
   }: RenewSubscriptionForUserProps): Promise<OptionalInvoice> {
     try {
-      const {periods = [], paidUntil} = userSubscription
+      const {periods = [], paidUntil, deactivatedAt} = userSubscription
       periods.sort((periodA, periodB) => {
         if (periodA.endsAt < periodB.endsAt) return -1
         if (periodA.endsAt > periodB.endsAt) return 1
         return 0
       })
       if (
-        periods.length > 0 &&
+        (periods.length > 0 || deactivatedAt === null) &&
```

Comment:
I think we don't need `> 0` part, what do you think?

Reply:
@bishoymelek why do you think that?